Sources/HTTP/SimpleSocket/SimpleSocketConnectionListener.swift

```diff
@@ -157,11 +165,7 @@ public class BlueSocketConnectionListener: ParserConnecting {
 
     /// Starts reading from the socket and feeding that data to the parser
     public func process() {
-        do {
-            try socket?.setBlocking(mode: true)
-        } catch {
-            fatalError("Failed to set socket as non-blocking")
-        }
+        socket?.setBlocking(mode: true)
```

Comment:
This ignores failure...

Reply:
Yeah. I'll do another pass with throwing.